src/hotspot/share/opto/superword.cpp

```diff
@@ -570,7 +570,7 @@ bool SuperWord::SLP_extract() {
     return false;
   }
 
-  extend_packlist();
```

Comment:
Note: `packlist` is the wrong name for `packset`. But to be more precise: we are extending the pairs (which are packs, and which are in the packset).